charts/connect/CHANGELOG.md

```diff
@@ -12,6 +12,15 @@
 
 ---
 
+[//]: # (START/v1.14.0)
+# v1.14.0
+
+## Features
+* The default Operator version is updated to v1.8.0 (kudos to @mmorejon) {#168}
```

Comment:
`[nitpick]`

We would like to have a consistent way of giving credit to external contributors. Check release for [connect-1.9.0](https://github.com/1Password/connect-helm-charts/releases/tag/connect-1.9.0) for the way we do it.

I will provide the suggestion here for simplicity as well: